README.rst

```diff
@@ -145,6 +153,13 @@ To build the user's guide::
 This will build HTML and PDF documentation, writing them to ``dist/userguide``
 and ``dist/InVEST_*_Documentation.pdf``, respectively.
 
+The User's Guide is maintained in a separate git reporsitory. InVEST will build
```

Comment:
It is admittedly fun to say `reporsitory`, but we should probably fix the typo anyway. 😃